bip-0085.mediawiki

```diff
@@ -370,16 +429,42 @@ The reason for running the derived key through HMAC-SHA512 and truncating the re
 
 Many thanks to Peter Gray and Christopher Allen for their input, and to Peter for suggesting extra application use cases.
 
+==Change Log==
```

Comment:
@jonatack i've added. real changelog so that the semvers are more... semantic. i could go deeper in terms of detail (fixes, etc.) but this seems complete enough to be useful and importantly puts this commit at semver 1.3.0.

Reply:
Suggest "Changelog" (no space), with entries ordered by most recent first (see https://keepachangelog.com/en/1.1.0/).